web-frontend/src/pages/TransferCoins/SendCoins/ContactList.tsx

```diff
@@ -7,16 +7,10 @@ import {
   MassaLogo,
 } from '@massalabs/react-ui-kit';
 import { formatStandard } from '../../../utils/massaFormat';
-import { useResource } from '../../../custom/api';
 import { AccountObject } from '../../../models/AccountModel';
 
 function AccountSelect({ ...props }) {
-  const { onClose, setRecipient, account: currentAccount } = props;
-
-  const { data: accounts = [] } = useResource<AccountObject[]>('accounts');
-  const filteredAccounts = accounts.filter(
-    (account: AccountObject) => account.nickname !== currentAccount.nickname,
-  );
+  const { onClose, setRecipient, filteredAccounts } = props;
```

Comment:
why are you renaming the prop account to filteredAccounts? if the filter is apply in the parent component, then this component is not aware of the filter, and so account is a good variable name

Reply:
i kept the same naming but it si true that it is better with accounts now :+1: